math/vdt/Portfile

```diff
@@ -24,11 +24,9 @@ license             LGPL-3+
 description         A collection of fast and inline implementations of mathematical functions.
 long_description    ${name} provides ${description}.
 
-platforms           darwin
-
 compiler.cxx_standard 2011
 
 # Disable extensions on Apple silicon for now
-if { ${configure.build_arch} eq "arm64" } {
+if { ${configure.build_arch} in [list arm64 ppc ppc64] } {
     configure.args-append   -DSSE=OFF -DNEON=OFF
```

Comment:
Doesn't ARM64 have NEON?

Reply:
I simply retained it for arm64 as it was in the port (no idea why it was that way). I can change it in a way you recommend, but without testing anything for arm.

Reply:
I don't know have the ability to test this on arm64 either, but it's peculiar. Doubtless disabling NEON makes it slower than it needs to be. I'd ask @cjones051073 if he knows why it is disabled.